src/overlays/actors/ovl_Bg_Spot03_Taki/z_bg_spot03_taki.h

```diff
@@ -4,11 +4,26 @@
 #include <ultra64.h>
 #include <global.h>
 
+typedef enum {
+    WATERFALL_CLOSED,
+    WATERFALL_OPENING_IDLE,
+    WATERFALL_OPENING_ANIMATED,
+    WATERFALL_OPENED,
+    WATERFALL_CLOSING,
+} BgSpot03TakiState;
+
 struct BgSpot03Taki;
 
+typedef void (*BgSpot03TakiActionFunc)(struct BgSpot03Taki*, struct GlobalContext*);
```

Comment:
```suggestion
typedef void (*BgSpot03TakiActionFunc)(struct BgSpot03Taki*, GlobalContext*);
```

Reply:
fixed but fyi there are a few files that make this same mistake